themes/default/views/editor/objects/screen_html.php

```diff
@@ -82,4 +82,49 @@
 
 	<div class="editorBottomPadding"><!-- empty --></div>
 	
-	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
\ No newline at end of file
+	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
+	<script type="text/javascript">
+		jQuery(function () {
+			var parse = function (val) {
+					if (typeof val === 'string') {
+						val = val.trim();
+						if (val){
+							var parts = val.match(/^\.?\s?(\d*)\s*(.*)$/);
+							parts.shift();
+							return parts;
+						}
+					}
+				return undefined;
+				};
+			jQuery('#idno_accession_series_and_parts').each(function () {
+				var $dot = jQuery('<span class="idno_dot">.</span>').hide(),
+					$series = jQuery('<input type="number" class="idno_accession_series" title="Enter series number" step="1" min="1"/>'),
+					$suffix = jQuery('<input type="text" class="idno_accession_suffix" size="10"/>'),
+					$this = jQuery(this).hide().before($dot).before($series).before($suffix),
+					seriesAndParts = parse($this.val()),
+					events = 'keypress change',
```

Comment:
OK I've just dev boxed this and in both firefox and chrome, it makes the dot appear or disappear based on the value _before_ the key being typed.  e.g. If the value is empty, then a digit is typed, the dot stays hidden.  Then another digit is typed, and the dot appears.  Then those two digits are both deleted but the dot remains, however when another digit is typed, the dot disappears.

Changing the `keypress` to `keyup` fixes this.  This is the thing that I was trying to remember about those events.